MiniApp/Classes/core/JavascriptBridge/MiniAppJavaScriptPermissionManager.swift

```diff
@@ -27,10 +27,15 @@ extension MiniAppScriptMessageHandler {
             miniAppPermissionRequestModelList.contains($0)
         }
 
-        let userNotRespondedRequestList = miniAppPermissionRequestModelList.filter {
+        var userNotRespondedRequestList = miniAppPermissionRequestModelList.filter {
             !allowedList.contains($0)
         }
 
+        /// To make sure that only permissions mentioned in the manifest is allowed at run-time
+        userNotRespondedRequestList = userNotRespondedRequestList.filter {
```

Comment:
I think we need to return the same permission list that was passed by the user. Just if they pass a permission which isn't in the manifest, then it gets set to DENIED.

So I think we shouldn't remove any permissions, just set to DENIED if it's not in the manifest.

Reply:
Done